src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,35 @@
+<script>
+	import fbLogo from './Facebook_Logo_Primary.png';
+	import igLogo from './Instagram_Glyph_Gradient.png';
+</script>
+
+<h1>Contact Us</h1>
+
+<p>
+	Email Address: <a href="mailto:kendoinfo@umich.edu"> kendoinfo@umich.edu</a>
+</p>
+
+<h2>Find us on:</h2>
+
+<a href="https://www.facebook.com/groups/1038072369705184">
+	<img class="social-media" src={fbLogo} alt="Facebook" />
+</a>
+<a href="https://www.instagram.com/kendoumich/">
+	<img class="social-media" src={igLogo} alt="Instagram" />
+</a>
+
+<style>
+	.social-media {
+		--icon-width: 75px;
+		min-width: 29px;
```

Comment:
What is the min-width for?

Reply:
Facebook press kit says their logo's size must be at minimum 16px wide/long (1:1 ratio). Instagram says 29px minimum. So I took the larger of the two.

Reply:
But 75px is more than 29px right?

Reply:
Yeah but I presumed when I start working on #21 "Style Pages" in earnest, we might change the width to be dynamic, in which case it has the potential to be sized smaller than 29px without the min width

Reply:
I mean yeah you can worry about that in 21 but we shouldn't be knowingly be adding zombie code.

Besides we don't actually want to set the minimum size of .social-media to 29px we want to set the minimum value of --icon-width to 29px

Reply:
Oh right. One of the problems with that is that if we just set the minimum size of .social-media then the padding calculation would be wrong. 
Resolved in [53c07b3](https://github.com/kendoclub-at-umich/website/pull/32/commits/53c07b3eb1a513a9a01e9015b96826df12289035)